tests/snapshots/assets__check_file@packages-fletcher-diagram.typ-80.snap

```diff
@@ -266,10 +266,9 @@ snapshot_kind: text
     error("Unexpected named argument(s) #..0.", args.named().keys())
   }
 
-  let positional-args = args
-    .pos()
-    .flatten()
-    .join() + [] // join to ensure sequence
+  let positional-args = (
+    args.pos().flatten().join() + []
+  ) // join to ensure sequence
   let objects = positional-args.children
```

Comment:
wait, that's weird. same as above

Reply:
caused by `+`

Reply:
Similarly, I'm not sure if we can only enter chain mode when the chain length is sufficiently large. So cases like `a+b` wouldn't look bad. I'm very curious about how rustfmt handle these cases

Reply:
wait. rust doesn't has weird code continue syntax so it doesn't need to handle this 😅 we may see how ruff/black handles this though

Reply:
![image](https://github.com/user-attachments/assets/2cb89dde-e697-4285-80a2-01d7b6c5b61d)

Reply:
> caused by `+`

Sorry but i don't get it. It looks like this line is far from 80 char limit. Is it possible to format it as following?

```
let positional-args = args
  .pos()
  .flatten()
  .join() + []
```

Reply:
Okay I think we've enter chain mode and we decide to put `+` on a new line. Therefore we will have to add parens.